tests/README.md

```diff
@@ -1,53 +1,21 @@
 # Test CI Job generation
 
-The CI jobs for these tests are managed by `build-workflows.py`.
+The CI jobs for these tests are managed by `build-job-matrix.py`.
+
+The test matrix is generated with a command line like the following:
+```
+python3 tests/build-job-matrix.py --per directory --verbose > matrix.json
+```
+
+The command to build the matrix lives in [test.yml](https://github.com/Chia-Network/chia-blockchain/tree/main/.github/workflows/test.yml)
 
-If you add a test file, or move one to another directory, please run `build-workflows.py`.
 Tests are recognized by the file glob `test_*.py`.
-Changing the contents of a file does not require running `build-workflows.py`.
 
 We currently use github actions. Default runners have two vcpus.
 The workflows are located in [../.github/workflows/](https://github.com/Chia-Network/chia-blockchain/tree/main/.github/workflows).
 
-The inputs to `build-workflows.py` are the templates in `runner-templates`, the file `testconfig.py` in this directory, and the optional `config.py` files in some test subdirectories.
-Files in the template directory ending in `include.yml` are included in jobs based on the per-directory settings.
-
-The generated workflows are output to `../.github/workflows/`.
+If the `--per directory` argument is used, each test subdirectory becomes a job in the github workflow matrix.
 
-Each subdirectory below the directories `root_test_dirs` in `testconfig.py` becomes a job in the github workflow matrix.
-If your jobs run too long, simply move some tests into new subdirectories and run `build-workflows.py`.
+If your jobs run too long, simply move some tests into new subdirectories and re-push your branch.
 A workflow built from a parent directory does not include the tests in its subdirectories.
 The subdirectory jobs do not include the tests from their parents.
-
```

Comment:
I would keep this whole section.  I think the only change is that parallel is now default?  In general it still applies though.